# request
uint32 id
LightControlType command
---
# response
LightControlType state